src/Commands/RunScan.spec.ts

```diff
@@ -67,4 +67,26 @@ describe('RunScan', () => {
       expect(act).toThrowError(SyntaxError);
     });
   });
+
+  describe('serializeWarnings', () => {
```

Comment:
From the business case described in #545 we can extract test case `should display warnings when present`. To test that we actually have to verify that `logger` with expected message and level was invoked. Please rework tests to cover `handle` method with the single test case mentioned above:

- Register [mock](https://github.com/NagRock/ts-mockito?tab=readme-ov-file#basics) `Scans` in container
- Do not forget to [reset](https://github.com/microsoft/tsyringe?tab=readme-ov-file#clearing-instances) it
- Arrange test data in test case
- Verify that expected log has been produced by [spying](https://github.com/NagRock/ts-mockito?tab=readme-ov-file#spying-on-real-objects) on `logger`